### PR TITLE
#1 add support for translations

### LINKED ISSUE #1: Create simple Hello world with a translation that uses PhraseApp

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
This closes #1

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    path: path.join(__dirname, 'dist'),
+    filename: `smart-faq.${language}.js`
+  },
+  module: {
+    rules: [
+      { test: /\.js$/, exclude: /node_modules/, loader: "babel-loader" }
+    ]
+  },
+  plugins: [
+    new CleanWebpackPlugin(['dist']),
```
Comment (oreqizer) [inline webpack.config.js]: a plugin for `rimraf`? 😄 i feel it's an overkill

File: src/i18next.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// @flow
```
Comment (oreqizer) [inline src/i18next.js]: is this file imported into actual production code?

Comment (jaroslav-kubicek) [reply src/i18next.js]: will be, but it will load only one resource for current language

File: i18next-scanner.config.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable import/no-extraneous-dependencies, no-console */
+// @noflow
+
+// copy pasted from https://github.com/i18next/i18next-scanner/blob/master/examples/i18next-scanner.config.js
```
Comment (oreqizer) [inline i18next-scanner.config.js]: oh my god this seems complicated for what it does

File: package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "dependencies": {
+    "i18next": "10.5.0",
+    "react": "16.2.0",
+    "react-i18next": "7.4.0"
+  },
+  "devDependencies": {
+    "babel-core": "6.26.0",
+    "babel-loader": "7.1.4",
+    "babel-preset-env": "1.6.1",
+    "clean-webpack-plugin": "0.1.18",
```
[the remark this answers is not in the document]
Comment (jaroslav-kubicek) [reply package.json]: fixed

**Pull request merged**